CGPs/cgp-0080.md

```diff
@@ -0,0 +1,130 @@
+
+Through the success of our initial proposal, certain areas are sufficiently addressed and thus have been deprioritized. For instance, Celo is now home to tokenized carbon assets from Toucan, FlowCarbon, Thallo, and several other projects; accordingly we will not prioritize funding of similar carbon tokenization protocols that may be redundant. 
+
+As discussed in our forum post, CC will arrive at a sustainable financing model before the end of the two year window supported by this proposal. 
+
+## Known Risks
+Informed by our last year of operations, we have identified the following risks:
+
+* Market risk: adverse events in crypto markets can severely reduce the value of funds available. This is mitigated by denominating the budget request in cUSD, and using a conservative price to account for CELO.
+* Competitive risk: certain other blockchain foundations are offering higher value grants. This is mitigated by CC’s past work in establishing a critical mass of leading climate projects on Celo.
+
+# Implementation
+
+## cUSD Denominated Budget
+For this new phase of Climate Collective, we ask to increase our spend permit on the community fund by 10MM CELO, for a total of 11.9MM CELO to be utilized over two years - <ins>**_but hope to not use the full amount_**</ins>. This number accounts for CELO at $0.42 and protects our budget from acute market volatility. Over 2 years, CC would make quarterly withdrawals of CELO equivalent to 625,000 cUSD based on the spot market price, and transact OTC with the help of Mento to convert this to cUSD (as opposed to selling large chunks of CELO on open markets). Any unutilized CELO will remain in the community fund.
+
+## Multisig
+The Multisig contract address is 0x275D5828bF7e5aaf51Bf8EF2f0668ac4a98F0d23 and can be found [here](https://safe.celo.org/mainnet:0x275D5828bF7e5aaf51Bf8EF2f0668ac4a98F0d23/home). Note this is the same Celo Safe multisig wallet used in our initial proposal. 
+
+The signers on this multisig are:
+* Anna Lerner: 0xA5489241e7b867FBCC646B6b2a6a31FD3C12d68f
+* Nirvaan Ranganathan: 0x42B4a02f79438D558AB33E326638ed752b94D187
+* Ed Walters: 0x3c4D00F7baebC752d366d2E2edD3f23BE2f6DF8e
+
+## Proposed Change
+```
+[
+  {
+    "contract": "GoldToken",
+    "function": "approve",
+    "args": [
+      "0x275D5828bF7e5aaf51Bf8EF2f0668ac4a98F0d23",
+      "11900000000000000000000000"
+    ],
+    "value": "0"
+  }
+]
+```
+## Verification
+
+Command
+celocli governance:show --proposalID 104 --node https://forno.celo.org
+
+You should see:
+```
+Running Checks:
+   ✔  104 is an existing proposal 
+proposal: 
+  0: 
+    contract: GoldToken
+    address: 0x471EcE3750Da237f93B8E339c536989b8978a438
+    function: approve
+    args: 
+      0: 0x275D5828bF7e5aaf51Bf8EF2f0668ac4a98F0d23
+      1: 11900000000000000000000000
```

Comment:
At the top of the proposal it mentions up to 10MM CELO but specifies a greater amount here